### PR TITLE
Unify BE-tasks

### PR DESCRIPTION
**Pull request by Gregoor**
@mikehenrty lmkwyt

### CODE REVIEW

File: server/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import Server from './server';
+
+const commands: any = {
+  test() {
+    require('./test/schema-migration');
+    require('./test/user-test');
```
[the remark this answers is not in the document]
Comment (Gregoor) [reply server/src/index.ts]: I'm with you on that, just didn't see a simple way to dynamically import them, but also letting webpack inline them.

File: server/src/test/lib/test-suite.ts [hunk 2]
```diff
@@ -57,11 +57,7 @@ class SuiteState {
    */
   private createTest(testCase: SuiteTestCase): void {
     const testName = `${this.name}::${testCase.name}`;
-    this.tape.test(
-      testName,
-      { timeout: TIMEOUT },
-      this.runTest.bind(this, testCase)
-    );
+    this.tape.test(testName, this.runTest.bind(this, testCase));
```
[the remark this answers is not in the document]
Comment (Gregoor) [reply server/src/test/lib/test-suite.ts]: Oh dang, forgot to mention that. It type errord, as `tape.test` only expects 2 parameters.

[the remark this answers is not in the document]
Comment (mikehenrty) [reply server/src/test/lib/test-suite.ts]: nice catch!

**Comment by Gregoor** [comment]
closing this in favor of just using Jest for testing

**Pull request closed without merging**